.travis.yml

```diff
@@ -21,6 +21,7 @@ script:
   - npm install
   - npm test -- ../
   - npm run build_all -- --style-dir=../ --conf=./prod_conf.json
+  - npm install semver
```

Comment:
`semver` should be installed in the parent folder, and preferably in the deploy script